app/uk/gov/hmrc/tariffclassificationfrontend/connector/StrideAuthConnector.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.connector
+
+import com.google.inject.Inject
+import javax.inject.Singleton
+import uk.gov.hmrc.auth.core.PlayAuthConnector
+import uk.gov.hmrc.http.CorePost
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+
+@Singleton
+class StrideAuthConnector @Inject()(configuration: AppConfig, client: HttpClient) extends PlayAuthConnector {
```

Comment:
Do we really need this connector?
If you use the library, this should not be needed

Reply:
The stride docs say to use it

Reply:
We either create it as a class or register an implementation with Guice. I figured this approach was better.